index.js

```diff
@@ -14,21 +14,21 @@ module.exports = {
    'extends': 'eslint:recommended',
 
    'plugins': [
-      'silvermine',
+      '@silvermine/eslint-plugin-silvermine',
    ],
 
    'rules': {
 
-      'silvermine/array-indentation': 'error',
-      'silvermine/call-indentation': 'error',
-      'silvermine/fluent-chaining': 'error',
-      'silvermine/no-multiple-inline-functions': 'error',
-      'silvermine/no-multiline-conditionals': 'error',
-      'silvermine/no-multiline-var-declarations': 'error',
-      'silvermine/indent': [ 'error', 3, { 'VariableDeclaratorOffset': { 'var': 1, 'let': 1, 'const': 3 }, 'SwitchCase': 1 } ],
-      'silvermine/empty-object-spacing': 'error',
-      'silvermine/empty-array-spacing': 'error',
-      'silvermine/uninitialized-last': 'error',
+      '@silvermine/silvermine/array-indentation': 'error',
```

Comment:
@jimjenkins5 Thanks for the [note about plugin naming](https://github.com/silvermine/eslint-plugin-silvermine/pull/23#discussion_r222406841) earlier! eslint is rather picky.